src/views/Events.vue

```diff
@@ -109,13 +109,14 @@ export default {
     },
 
     //unixToDate: converts the given unix timestamp to a javascript date number
+    //legacy code
```

Comment:
There's really zero reason to have "legacy code" in a code base that is currently under development. If this method does nothing then just remove it